### PR TITLE
Make it more clear that `buildozer` accepts multiple commands

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

File: buildozer/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ go install github.com/bazelbuild/buildtools/buildozer@latest
 ## Usage
 
 ```shell
-buildozer [OPTIONS] ['command args' | -f FILE ] label-list
+buildozer [OPTIONS] ['command args'... | -f FILE ] label-list
```
Comment (aiuto) [inline buildozer/README.md:18]: Did you mean `'command args'...`  or   `'command args' ...`

Comment (fmeum) [reply buildozer/README.md:18]: I want to "match" `'command args'`, `'command args' 'command args'`, .... Not entirely sure what the typical conventions for help texts are, what would you say?

Comment (laurentlb) [reply buildozer/README.md:18]: https://stackoverflow.com/a/9727046

The PR looks good to me

**Review by laurentlb — APPROVED** [review]

**Comment by aiuto** [comment]
Is anyone going to merge?

**Review by vladmos — APPROVED** [review]

**Pull request merged**